CONTRIBUTING.md

```diff
@@ -182,39 +182,39 @@ Follow the steps to get your work included in the project.
    ```bash
    git checkout -b john/138/buckets-undefined-index
    ```
-   
+
    Or
-   
+
    ```bash
    git checkout -b john/fix/138
    ```
 
-
-4. It's coding time!
+4) It's coding time!
    Please respect the coding convention: [Coding guidelines](https://github.com/fossology/fossology/wiki/Coding-Style)
-   
+
    Commit your changes in logical chunks. Please adhere to **Git Commit Conventions**
    and [Coding guidelines](https://github.com/fossology/fossology/wiki/Coding-Style)
    or your code is unlikely be merged into the main project.
-   Use Git's [interactive rebase](https://help.github.com/articles/interactive-rebase)
+   Use Atlassian's [interactive rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)
```

Comment:
The interactive rebase is feature of git but the guide is from Atlassian. So the changing `Git's` -> `Atlassian's` will be wrong.